src/style.css

```diff
@@ -0,0 +1,26 @@
+html,
+body {
+  font-family: Arial, sans-serif;
+  font-size: 13px;
+}
+
+h1,
+h2,
+p {
+  margin: 0;
+}
```

Comment:
Try to avoid stylizing tags. Always use classes for this. Tags stylizing is OK when you reset default browser styles, like margin: 0 for BODY tag, which has 8px margins. Setting width and height for all DIV tags is bad practice

Reply:
посмотрите сегодняшнее занятие, Миша сказал, что сброс параметров заголовков и параграфов было конкретно указано в задании, согласуйте как-то уроки и проверки

Reply:
you could use body for resetting margins, not three separate tags